modules/ROOT/pages/managing-users.adoc

```diff
@@ -91,6 +91,67 @@ If enabled, the SAML assertions sent from the IdP must be encrypted and follow t
 +
 . Sign out of Anypoint Platform, navigate to the sign-on URL you entered in the *Identity Management SAML 2.0* form, then sign in through your identity provider to test the configuration.
 
+== Rotate a SAML Key
+
+If your IdP supports multiple keys, you can rotate keys to seamlessly retire keys that have been in use for an extended period of time.
```

Comment:
"As a security best practice, many organizations like to rotate the key used to encrypt SAML 2.0 assertions.  With the key rotation feature, Anypoint Platform can generate a new key, or you can upload a public and private key pair (up to 3).  

After you've added a new key to Anypoint Platform, you must ensure it exists in your IdP.   All keys present in Anypoint Platform will be used to decrypt SAML 2.0 assertions sent by your IdP.  However, only the `primary` key will be used by Anypoint Platform to sign SAML 2.0 AuthNRequests and SLO requests and must by configured in your IdP to validate the signatures.

If you are migrating from the default Anypoint Platform SSO certificate to a new certificate, you must update the Assertion Consumer Service (ACS) URL in your IdP.  Only the new ACS URL supports the keys generated by the key rotation feature. If your ACS URL already follows the pattern of `.../accounts/login/:org-domain/providers/:providerId/receive-id`, you do not need to change it.